src/main/scala/com/indix/mesos/GOCDPoller.scala

```diff
@@ -28,19 +52,48 @@ case class GOCDPoller(conf: FrameworkConfig) {
   }
 
   def goIdleAgentsCount() = {
+    println("Polling Go server for idle agents")
+    try {
+      val responseJson = jsonRequest(buildUrl() + "/go/api/agents")
```

Comment:
@ashwanthkumar The difference in the responses are because of the Accept header we send. The difference is this:

`curl -X GET http://build.force.io/go/api/agents -H "Authorization: Basic asdadsadasdasd="`

and 

`curl -X GET http://build.force.io/go/api/agents -H "Authorization: Basic adsdasdasda=" -H "Accept: application/vnd.go.cd.v1+json"`

:)